app/models/explainer/explained_query.rb

```diff
@@ -18,6 +18,10 @@ def indexed?
       type == 'ref' && key != NULL
     end
 
+    def full_table_scan?
+      extra == 'Using where'
```

Comment:
I don't think this implies a full table scan https://dev.mysql.com/doc/refman/5.7/en/table-scan-avoidance.html